cosmo_tester/framework/cluster.py

```diff
@@ -634,15 +641,18 @@ def _clone_manager_blueprints(self):
 
     def _create_inputs_file(self):
         self._inputs_file = self._tmpdir / 'inputs.json'
-        bootstrap_inputs = json.dumps({
+        bootstrap_inputs = {
             'public_ip': self.managers[0].ip_address,
             'private_ip': self.managers[0].private_ip_address,
             'ssh_user': self._attributes.centos7_username,
             'ssh_key_filename': self._ssh_key.private_key_path,
             'admin_username': self._attributes.cloudify_username,
             'admin_password': self._attributes.cloudify_password,
-            'manager_resources_package': self._manager_resources_package},
-            indent=2)
+        }
+        if self._manager_resources_package:
```

Comment:
Ah- I wonder if this is the cause of the community bootstrap test problems- I'll have a poke.